### DISCUSSION TITLE
Svelkit CORS...

### DISCUSSION THREAD

**Discussion by Dredious** [vercel/community#3055] [Help]
### Summary

Basic App deployed to Vercel.

### Steps to Reproduce

Basic App deployed to Vercel, with 2 API routes both of which run into CORS. I have followed the vercel.json to no avail.
Have seen many people in the same boat, has this been sorted?

**Comment by amyegan** [comment]
Hi @Dredious! You can use the [json convenience function](https://kit.svelte.dev/docs/modules#sveltejs-kit-json) to set access control [headers](https://kit.svelte.dev/docs/web-standards#fetch-apis-headers) on the response in SvelteKit. You can also use [hooks](https://kit.svelte.dev/docs/hooks#server-hooks) to add headers to all `/api` routes. 

More information can be found on these pages:
- [Fetch APIs: Headers](https://kit.svelte.dev/docs/web-standards#fetch-apis-headers)
- [Hooks](https://kit.svelte.dev/docs/hooks#server-hooks)
- [Loading data: Cookies and Headers](https://kit.svelte.dev/docs/load#cookies-and-headers)
